### ISSUE TITLE
Extending webpack config fails without user-provided `module` property

### ISSUE THREAD

**Issue by fvgs** [storybooks/storybook#1467]
Running `start-storybook` after adding my own `webpack.config.js` produces the following error:

Using **v3.1.7**
```
rules: [].concat((0, _toConsumableArray3.default)(config.module.rules),
(0, _toConsumableArray3.default)(customConfig.module.rules || []))

TypeError: Cannot read property 'rules' of undefined
```

webpack.config.js
```javascript
const webpack = require('webpack');

module.exports = {
  plugins: [
    new webpack.ProvidePlugin({
      $: 'jquery',
    }),
  ],
};
```

Storybook accesses `customConfig.module` which may be undefined if the user webpack config does not contain a `module` property. Storybook further attempts to access `customConfig.module.rules` which, in the case described, results in the above error.

**Comment by Ethaan** [comment]
Did you fix this?

**Comment by fvgs** [comment]
@Ethaan yes, this was fixed in #1468
